examples/resnet-18/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers
```

Comment:
Maybe worth considering moving resnet from examples to python/turbine_models in some form to enable use outside turbine?